WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/LandingPage.kt

```diff
@@ -0,0 +1,40 @@
+package org.wordpress.android.e2e.pages
+
+import androidx.compose.ui.test.junit4.ComposeTestRule
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import org.wordpress.android.BuildConfig
+import org.wordpress.android.R
+import org.wordpress.android.support.WPSupportUtils.clickOn
+import org.wordpress.android.support.WPSupportUtils.getTranslatedString
+import org.wordpress.android.ui.pages.LandingPageBase
+
+object LandingPage {
+    private const val isNewUiEnabled = BuildConfig.IS_JETPACK_APP || BuildConfig.LANDING_SCREEN_REVAMP
+
+    @JvmStatic
+    fun tapContinueWithWpCom(composeTestRule: ComposeTestRule) {
+        if (isNewUiEnabled) {
+            // New UI - See LoginPrologueRevampedFragment
+            composeTestRule
+                .onNodeWithText(getTranslatedString(LandingPageBase.continueWithWpComButtonStringRes))
```

Comment:
What do you think about using `testTag` instead for this?

```suggestion
                .onNodeWithTag("continueButton")
```

If we use `testTag`, it should continue to work on both Jetpack and WordPress and we wouldn't have to introduce these new files to store the strings:
`WordPress/src/androidTestJetpack/java/org/wordpress/android/ui/pages/LoginPage.kt`
`WordPress/src/androidTestWordpress/java/org/wordpress/android/ui/pages/LandingPageBase.kt`

The testTag can be added as part of the `PrimaryButton`, on Jetpack: https://github.com/wordpress-mobile/WordPress-Android/blob/06b4658163739ac0f4809c7c9291fa5d3f649e5f/WordPress/src/jetpack/java/org/wordpress/android/ui/accounts/login/components/PrimaryButton.kt#L34-L37

and on WordPress:
https://github.com/wordpress-mobile/WordPress-Android/blob/06b4658163739ac0f4809c7c9291fa5d3f649e5f/WordPress/src/wordpress/java/org/wordpress/android/ui/accounts/login/LoginPrologueRevampedFragment.kt#L109-L112

WDYT?

Reply:
Thank you @jostnes 🙇 👍 

> I did notice a few issues for some of the Jetpack tests when running locally, but I think those are out of the scope of this PR and can be fixed separately, as Jetpack tests are still local only at the moment.

Glad you're mentioning it and agree to keep the scope of this PR limited at **adding the Compose layout testing support and the basic support for login with JP app in tests** 🙏 

> What do you think about using `testTag` instead for this?
> If we use `testTag`, it should continue to work on both Jetpack and WordPress and we wouldn't have to introduce these new files to store the strings [...]

I briefly learned about `testTag` from [Testing your Compose layout](https://developer.android.com/jetpack/compose/testing) before starting on this PR.

For some reason I hesitated to explore this approach because it requires adapting the implementation code for testing which at least when it's about unit testing isn't recommended; although if we think about the `testability` term then of course good code should be testable.

I like your suggestion to use `testTag` and I will explore it 🙇 , I also like the fact that it reminds me of SwiftUI's `accessibilityIdentifier` 😄 
Thanks a lot for your input 👍

Reply:
Works like a charm 💯 🚀!
See commit: https://github.com/wordpress-mobile/WordPress-Android/pull/17869/commits/7a4ba84d010e6b2a830f5b6ad811bbffe030fc2a